data_model/src/transaction.rs

```diff
@@ -98,7 +98,7 @@ mod model {
         Serialize,
         IntoSchema,
     )]
-    pub(crate) struct TransactionPayload {
+    pub struct TransactionPayload {
```

Comment:
the same as [here](https://github.com/hyperledger/iroha/pull/4456/files#r1568427689)